modules/create-policy-from-risk-view.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: PROCEDURE
+[id="create-policy-from-risk-view_{context}"]
+= Creating a security policy from the risk view
+
+[role="_abstract"]
+While evaluating risks in your deployments in the *Risk* view, when you apply local page filtering, you can create new security policies based on the filtering criteria you are using.
+
+.Procedure
+. Navigate to the RHACS portal and select *Risk* from the left-hand navigation menu.
+. Apply local page filtering criteria for which you want to create a policy.
```

Comment:
```suggestion
. Apply local page filtering criteria that you want to create a policy for.
```

ISG: Use a preposition at the end of a sentence to avoid an awkward or stilted construction.